pkg/collection/entry/entry_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package entry_test
+
+import (
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/collection"
+	"github.com/ethersphere/bee/pkg/collection/entry"
+	"github.com/ethersphere/bee/pkg/swarm/test"
+)
+
+var (
+	_ = collection.Entry(&entry.Entry{})
```

Comment:
To ensure interface implementation during build, this line should not be in test file, but in entry.go file, as build is ignoring test files.